tgui/packages/tgui/interfaces/Hypertorus.js

```diff
@@ -49,296 +192,229 @@ export const Hypertorus = (props, context) => {
+              value={internal_fusion_temperature}
+              minValue={0}
+              maxValue={1e30}>
+              {formatSiBaseTenUnit(internal_fusion_temperature * 1000, 1, 'K')}
+            </ProgressBar>
+          </LabeledList.Item>
+          <LabeledList.Item label="Moderator gas temperature">
+            <ProgressBar
+              color={'red'}
+              value={moderator_internal_temperature}
+              minValue={0}
+              maxValue={1e30}>
+              {formatSiBaseTenUnit(moderator_internal_temperature * 1000, 1, 'K')}
+            </ProgressBar>
+          </LabeledList.Item>
+          <LabeledList.Item label="Output gas temperature">
+            <ProgressBar
+              color={'pink'}
+              value={internal_output_temperature}
+              minValue={0}
+              maxValue={1e30}>
+              {formatSiBaseTenUnit(internal_output_temperature * 1000, 1, 'K')}
+            </ProgressBar>
+          </LabeledList.Item>
+          <LabeledList.Item label="Coolant output temperature">
+            <ProgressBar
+              color={'green'}
+              value={internal_coolant_temperature}
+              minValue={0}
+              maxValue={1e30}>
+              {formatSiBaseTenUnit(internal_coolant_temperature * 1000, 1, 'K')}
+            </ProgressBar>
+          </LabeledList.Item>
+        </LabeledList>
+      </Section>
+    </>
+  );
+};
+
+const HypertorusTabs = (props, context) => {
+  const { act, data } = useBackend(context);
+  const [
+    tabIndex,
+    setTabIndex,
+  ] = useLocalState(context, 'tab-index', 1);
+  return (
+    <>
+      <Tabs>
+        <Tabs.Tab
+          selected={tabIndex === 1}
+          onClick={() => {
+            setTabIndex(1);
+          }}>
+          HFR Main controls
+        </Tabs.Tab>
+        <Tabs.Tab
+          selected={tabIndex === 2}
+          onClick={() => {
+            setTabIndex(2);
+          }}>
+          HFR Secondary controls
+        </Tabs.Tab>
+        <Tabs.Tab
+          selected={tabIndex === 3}
+          onClick={() => {
+            setTabIndex(3);
+          }}>
+          HFR internal gases
+        </Tabs.Tab>
+        <Tabs.Tab
+          selected={tabIndex === 4}
+          onClick={() => {
+            setTabIndex(4);
+          }}>
+          HFR internal parameters
+        </Tabs.Tab>
+      </Tabs>
+      {tabIndex === 1 && (
+        <HypertorusMainControls />
+      )}
+      {tabIndex === 2 && (
+        <HypertorusSecondaryControls />
+      )}
+      {tabIndex === 3 && (
+        <HypertorusGases />
+      )}
+      {tabIndex === 4 && (
+        <HypertorusParameters />
+      )}
+    </>
+  );
+};
+
+
+export const Hypertorus = (props, context) => {
+  const { act, data } = useBackend(context);
+  return (
+    <Window
+      title="Fusion Reactor"
+      width={500}
+      height={600}>
+      <Window.Content scrollable>
+        <Section title="Hypertorus Fusion Reactor control panel">
```

Comment:
Remove this section, because it is redundant and all sections inside are not visibly separated because of it.

Reply:
I'll make this one as title